### PR TITLE
added assertFinite, assertInfinite and assertNan

### PR DESCRIPTION
**Pull request by tronsha**

### CODE REVIEW

**Comment by sebastianbergmann** [comment]
Merged manually, thanks.

**Comment by sebastianbergmann** [comment]
Would be great if you could document (https://github.com/sebastianbergmann/phpunit-documentation/issues/308) these new assertions, @tronsha.

File: src/Framework/Assert.php [hunk 1]
```diff
@@ -968,6 +968,42 @@ public static function assertNull($actual, $message = '')
...
+    /**
+     * Asserts that a variable is nan.
+     *
+     * @param mixed  $actual
+     * @param string $message
+     */
+    public static function assertNan($actual, $message = '')
+    {
+        self::assertThat($actual, self::isNan(), $message);
+
```
Comment (GrahamCampbell) [inline src/Framework/Assert.php]: cs

Comment (sebastianbergmann) [inline src/Framework/Assert.php]: Was already fixed in 182be7c2e78b7f05f2416e833e9a0b2815fbacc3

**Pull request closed without merging**